service-clients-gen/src/main/java/com/flipkart/poseidon/serviceclients/generator/Generator.java

```diff
@@ -88,14 +88,24 @@ private static boolean validateInput(String[] args) {
 
         determineVersion(args[1]);
         String majorVersion = "v" + version.getMajor();
-        packageName = PACKAGE_NAME + moduleName + "." + majorVersion;
+
+        String serviceClientGroupId = System.getProperty("service.clients.groupId");
```

Comment:
thanks for the PR. What's the use case to have custom group id for generated service clients?  

@shrey-garg , can you also review this please?

Reply:
@pmohankumar by default service clients generated by this Generator group them in package **com.flipkart.poseidon.serviceclients**. As it is an open-source project, developers should have the flexibility to provide their own package group name as per their needs.
This change will let them use their own group id e.g. **com.companyname.serviceclients** instead of the default one.
In the case of groupId is not provided it will use the default groupId as **com.flipkart.poseidon.serviceclients**

Usage: 
`mvn clean install -Dservice.clients.groupId=com.companyname.serviceclients`

Reply:
sure, makes sense